pkg/ocicni/types.go

```diff
@@ -32,6 +32,16 @@ type NetworkConfig struct {
 	IP string
 }
 
+type BandwidthConfig struct {
+	// IngressRate is a limit for incoming traffic
```

Comment:
Can you specify the units that these are in? eg I believe it's bytes-per-second per CNI conventions. Also, can we do these as uint64s?

Reply:
done :)